ep/build-velox/src/get_velox.sh

```diff
@@ -242,8 +230,16 @@ if [[ "$RUN_SETUP_SCRIPT" == "ON" ]]; then
     echo "Unsupported kernel: $OS"
     exit 1
   fi
+}
+
+prepare_velox_source_code
+
+if [[ "$RUN_SETUP_SCRIPT" == "ON" ]]; then
```

Comment:
Actually, I don't think it's necessary to pass `RUN_SETUP_SCRIPT` here. If we have a brand new velox, we should just apply the setup fixes as we apply other patches or fixes. (Once is enough)

Then the user can repeatedly execute .builddeps-veloxbe.sh setup_dependencies, because setup generally needs to be executed repeatedly in a new environment (my experience)